TLDR.md

```diff
@@ -0,0 +1,15 @@
+# TLDR for the TLDR
+
+We're building a secure, decentralized communication protocol. Protocol v1.0.0 Jasper has multiple REST API servers in a mesh network. Clients connect to different TX (transmission) and RX (reception) servers. Servers can detect and report misbehaving servers. Everything is signed by users' keys.
+
+# Introduction
+
+Hey! This is Purple Bubble. We're a crew of teens on a mission to make messaging epic again. Zero tracking, spying, or data selling. Everything is end-to-end encrypted, anonymized, and decentralized. All open source.
+
+> **Wanna chat?**
+>
+> We mostly communicate through #purplebubble in the [Hack Club Slack](hack.club/slack), but since Hack Club is running a (super cool, you should check it out if you're a teen or younger) [summer event](hack.club/arcade), you will need to verify that you're not an adult to join. Worry not! You can always reach out to team@purplebubble.org or open a github issue.
```

Comment:
Can you update this to use the purple bubble slack and remove hack club references pls